src/exoscale/compute/api/http.clj

```diff
@@ -128,19 +110,14 @@
         (d/chain (json-request!! config opcode paged-params)
                  (list-pager-fn opcode page result))))))
 
-(defn job-result
-  [opcode {:keys [jobresult]}]
-  (let [[_ entity] (action-entity opcode)]
-    (or (get jobresult entity) jobresult)))
-
 (defn wait-or-return-job!!
   [config remaining opcode]
   (let [interval (or (:poll-interval config) default-poll-interval)]
     (fn [{:keys [jobstatus] :as job}]
       (if (zero? jobstatus)
         (d/chain (t/in interval (constantly nil))
                  (fn [_] (d/recur (dec remaining))))
-        (job-result opcode job)))))
+        (-> job :jobresult (dissoc :count) vals first)))))
```

Comment:
```suggestion
        (-> job :jobresult vals first)))))
```

(it doesn't hurt, but shouldn't be needed)

Reply:
thanks!